corehq/messaging/scheduling/forms.py

```diff
@@ -2951,7 +2989,13 @@ def add_initial_for_custom_metadata(self, result):
             len(self.initial_schedule.custom_metadata) > 0
         ):
             result['capture_custom_metadata_item'] = self.YES
-            for name, value in self.initial_schedule.custom_metadata.items():
+            custom_metadata = self.initial_schedule.custom_metadata
+
+            default_stale_interval_in_days = int(settings.SMS_QUEUE_STALE_MESSAGE_DURATION / 24)
+            sms_stale_after = custom_metadata.pop('sms_stale_after', default_stale_interval_in_days)
+            result['sms_stale_after'] = sms_stale_after
```

Comment:
why do we need this? won't the form auto populate the values if present and else leave it blank?

Reply:
It will. I added it as it will give an idea of default value to user.

Reply:
but then it will also get saved on the alert as the value? we want that? and hence then it would get added on all SMS's custom metadata too?

Reply:
thinking of the cases where the user does not want to add any value to this, which would be almost all the alerts

Reply:
> hence then it would get added on all SMS's custom metadata too?

No, it will not. I have explicitly added a check which ensures that default values are not saved.
https://github.com/dimagi/commcare-hq/pull/28419/files#diff-594ca011fb9871d019f20b229abad83aR3561

Reply:
I think it would be better to just leave it blank and just have a help note that tells what the default value is.

Reply:
nit: can get rid of `sms_stale_after` variable and assign directly instead